doc/source/serve/deployment-graph-cookbook.md

```diff
@@ -0,0 +1,60 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.6
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(serve-deployment-graph-cookbook)=
+
+# Deployment Graph Cookbook
+
+### Chain Nodes
+
+The example shows how to chain nodes using the same class and having different args passed in
+
++++
+
+```python
+import ray
+from ray import serve
+from ray.experimental.dag.input_node import InputNode
+from ray.serve.drivers import DAGDriver
+
+ray.init(num_cpus=16)
+serve.start()
+
+@serve.deployment
+class Model:
+   def __init__(self, weight):
+      self.weight = weight
+   def forward(self, input):
+      return input +  self.weight
+
+
+# 10 nodes chain
+num_nodes = 10
+nodes = [Model.bind(w) for w in range(num_nodes)]
+prev_outputs = [None] * num_nodes
```

Comment:
i think it's worth having a "warning" as "anti-pattern" section for this part explaining why using a variable name and re-assign it in a loop is not the right way.